plugins/fritz-box7490/de/fritzbox7490.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/bash
+# FritzBox.sh
+
+### Quelle des Codekerns - Target of die skript core ############################
+# https://github.com/Tscherno/Fritzbox.sh                                       #
+# compatible with Fritz.box Firmware 6.50 and higher, Horst Schmid, 2016-01-05  #
+# /usr/local/addons/cuxd/user/FritzBox.sh                                       # 
+#################################################################################
```

Comment:
Please explain, what the plugin does and how it works (e.g. does it run on the router or on a host in the local network). A link to an unreliable remote location is not sufficient.
You also need to mention all requirements (e.g. curl).